src/renderer/App.js

```diff
@@ -104,11 +112,24 @@ const App = (props) => {
     await navigate("/keyboard-select");
   };
 
+  const handleNativeThemeUpdate = (event, v) => {
+    if (theme != "system") return;
+
+    // This enforces a re-render, without having to use another state.
+    setTheme(null);
+    setTheme("system");
+  };
+
   useEffect(() => {
     ipcRenderer.on("usb-device-disconnected", handleDeviceDisconnect);
+    ipcRenderer.on("native-theme/updated", handleNativeThemeUpdate);
```

Comment:
the /updated convention is fine, but if we’re going to do that we should update our other ipc endpoints to match the style.

Reply:
I'm not quite sure what would be the best convention. At the moment we have `some-thing-event`, `some-thing/event` and `some-thing.event` too.

I'd rather change this PR to match one of the existing ones (`some-thing-event` or `some-thing.event`), than update all the rest. We should probably come up with some guidelines for IPC endpoints at some point, but lets tackle that in its own issue.

I'll update this PR to use `some-thing-event` meanwhile.

Reply:
I'm ok with "not introduce a new convention" as the bar for this PR :)